clients/python/tests/test_client.py

```diff
@@ -85,14 +84,18 @@ async def test_async_inference_streaming(async_client):
         tags={"key": "value"},
         stream=True,
     )
-    first_chunk_duration = None
+
     chunks = []
+    previous_chunk_timestamp = None
+    last_chunk_duration = None
     async for chunk in stream:
+        if previous_chunk_timestamp is not None:
+            last_chunk_duration = time() - previous_chunk_timestamp
+        previous_chunk_timestamp = time()
         chunks.append(chunk)
-        if first_chunk_duration is None:
-            first_chunk_duration = time() - start_time
-    last_chunk_duration = time() - start_time - first_chunk_duration
-    assert last_chunk_duration > first_chunk_duration + 0.1
+
+    assert last_chunk_duration > 0.01
+
     expected_text = [
```

Comment:
am I correct in that we were already testing this then?

Reply:
yeah, but somewhat incorrectly (we were checking the duration of all chunks after the first combined)